<html>
<head>
    <meta charset="utf-8"/>
<meta name="description" content=""/>
<meta name="viewport" content="width=device-width, initial-scale=1"/>

<title>蛋白设计相关知识 | </title>

<link rel="shortcut icon" href="https://justindoit.github.io/favicon.ico?v=1626016521702">

<link href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free/css/all.min.css" rel="stylesheet">
<link rel="stylesheet" href="https://justindoit.github.io/styles/main.css">
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.0/dist/css/bootstrap.min.css">

<script src="https://cdn.jsdelivr.net/npm/@highlightjs/cdn-assets/highlight.min.js"></script>
<script src="https://cdn.bootcss.com/highlight.js/9.15.10/languages/dockerfile.min.js"></script>
<script src="https://cdn.bootcss.com/highlight.js/9.15.10/languages/dart.min.js"></script>

<script src="https://cdn.jsdelivr.net/npm/moment@2.27.0/moment.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/jquery@3.5.1/dist/jquery.slim.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.0/dist/js/bootstrap.min.js"></script>
<!-- DEMO JS -->
<!--<script src="media/scripts/index.js"></script>-->



    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.css">
</head>
<body>
<div class="main gt-bg-theme-color-first">
    <nav class="navbar navbar-expand-lg">
    <div class="navbar-brand">
        <img class="user-avatar" src="/images/avatar.png" alt="头像">
        <div class="site-name gt-c-content-color-first">
            
        </div>
    </div>
    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
        <i class="fas fa-bars gt-c-content-color-first" style="font-size: 18px"></i>
    </button>
    <div class="collapse navbar-collapse" id="navbarSupportedContent">
        <div class="navbar-nav mr-auto" style="text-align: center">
            
                <div class="nav-item">
                    
                        <a href="/archives" class="menu gt-a-link">
                            列表
                        </a>
                    
                </div>
            
                <div class="nav-item">
                    
                        <a href="/" class="menu gt-a-link" target="_blank">
                            首页
                        </a>
                    
                </div>
            
                <div class="nav-item">
                    
                        <a href="/tags" class="menu gt-a-link">
                            标签
                        </a>
                    
                </div>
            
                <div class="nav-item">
                    
                        <a href="/post/about" class="menu gt-a-link">
                            关于
                        </a>
                    
                </div>
            
        </div>
        <div style="text-align: center">
            <form id="gridea-search-form" style="position: relative" data-update="1626016521702" action="/search/index.html">
                <input class="search-input" autocomplete="off" spellcheck="false" name="q" placeholder="搜索文章" />
                <i class="fas fa-search gt-c-content-color-first" style="position: absolute; top: 9px; left: 10px;"></i>
            </form>
        </div>
    </div>
</nav>

    <div class="post-container">
        <div class="post-detail gt-bg-theme-color-second">
            <article class="gt-post-content">
                <h2 class="post-title">
                    蛋白设计相关知识
                </h2>
                <div class="post-info">
                    <time class="post-time gt-c-content-color-first">
                        · 2021-04-16 ·
                    </time>
                    
                </div>
                <div class="post-content">
                    <ul>
<li>Reviews</li>
<li>General</li>
<li>Model repositories and resources</li>
<li>Chemoinformatics
<ul>
<li>fingerprints</li>
</ul>
</li>
<li>drug discovery</li>
<li>Biomarker discovery</li>
<li>Generic 'omics tool</li>
<li>Proteomics</li>
<li>Metabolomics</li>
<li>Generative models</li>
<li>Genomics<br>
+ Variant calling<br>
+ Gene expression<br>
+ Imaging and gene expression<br>
+ Predicting enhancers and regulatory regions<br>
+ Non-coding RNA<br>
+ Methylation<br>
+ Single-cell applications<br>
+ Population genetics<br>
+ Systems biology</li>
<li>Neuroscience</li>
</ul>
<h2 id="计算化学">计算化学</h2>
<h3 id="protein-homology-detection-without-alignment">protein homology detection (without alignment)</h3>
<ul>
<li>
<p><strong>2007-08</strong> | Fast model-based protein homology detection without alignment | <em>Sepp Hochreiter, Martin Heusel, and Klaus Obermayer</em> | [Bioinformatics](https://doi.org/10.1093/bioinformatics/btm247</p>
</li>
<li>
<p><strong>2012-07</strong> | Deep architectures for protein contact map prediction | <em>Pietro Di Lena, Ken Nagata and Pierre Baldi</em> <a href="https://doi.org/10.1093/bioinformatics/bts475">Bioinformatics</a></p>
</li>
<li>
<p><strong>2012-10</strong> | Predicting protein residue–residue contacts using deep networks and boosting | <em>Jesse Eickholt and Jianlin Cheng</em> | <a href="https://doi.org/10.1093/bioinformatics/bts598">Bioinformatics</a></p>
</li>
<li>
<p><strong>2013-03</strong> | DNdisorder: predicting protein disorder using boosting and deep networks | <em>Jesse Eickholt and Jianlin Cheng</em> | <a href="https://doi.org/10.1186/1471-2105-14-88">BMC Bioinformatics</a></p>
</li>
<li>
<p><strong>2014-06</strong> | Deep learning of the tissue-regulated splicing code | <em>Michael K. K. Leung, Hui Yuan Xiong, Leo J. Lee and Brendan J. Frey</em> | <a href="https://doi.org/10.1093/bioinformatics/btu277">Bioinformatics</a></p>
</li>
<li>
<p><strong>2014-10</strong> | DANN: a deep learning approach for annotating the pathogenicity of genetic variants  | <em>Daniel Quang, Yifei Chen and Xiaohui Xie</em> | <a href="https://doi.org/10.1093/bioinformatics/btu703">Bioinformatics</a></p>
</li>
<li>
<p><strong>2014-11</strong> | Pairwise input neural network for target-ligand interaction prediction | <em>Caihua Wang, Juan Liu, Fei Luo, Yafang Tan, Zixin Deng, Qian-Nan Hu</em> | <a href="https://doi.org/10.1109/BIBM.2014.6999129">2014 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</a></p>
</li>
<li>
<p><strong>2014-12</strong> | Deep learning as an opportunity in virtual screening | <em>Thomas Unterthiner, Andreas Mayr, Günter Klambauer, Marvin Steijaert, Jörg K. Wegner, Hugo Ceulemans, &amp; Sepp Hochreiter</em> | <a href="http://www.datascienceassn.org/sites/default/files/Deep%20Learning%20as%20an%20Opportunity%20in%20Virtual%20Screening.pdf">In Proceedings of the Deep Learning Workshop at NIPS</a>.</p>
</li>
<li>
<p><strong>2015-01</strong> | Unsupervised feature construction and knowledge extraction from genome-wide assays of breast cancer with denoising autoencoders. | <em>Jie Tan, Matt Ung, Chao Cheng, Casey Greene</em> | <a href="https://doi.org/10.1142/9789814644730_0014">Pacific Symposium on Biocomputing (PSB)</a> | <a href="http://discovery.dartmouth.edu/~cgreene/da-psb2015/">Models &amp; Data</a></p>
</li>
<li>
<p><strong>2015-01</strong> | The human splicing code reveals new insights into the genetic determinants of disease  | <em>Hui Y. Xiong, Babak Alipanahi, Leo J. Lee, Hannes Bretschneider, Daniele Merico, Ryan K. C. Yuen, Yimin Hua, Serge Gueroussov, Hamed S. Najafabadi, Timothy R. Hughes, Quaid Morris, Yoseph Barash, Adrian R. Krainer, Nebojsa Jojic, Stephen W. Scherer, Benjamin J. Blencowe, Brendan J. Frey</em> | <a href="https://doi.org/10.1126/science.1254806">Science</a></p>
</li>
<li>
<p><strong>2015-03</strong> | Deep Feature Selection: Theory and Application to Identify Enhancers and Promoters | <em>Yifeng Li, Chih-Yu Chen, and Wyeth W. Wasserman</em> | <a href="https://doi.org/10.1007/978-3-319-16706-0_20">19th Annual International Conference, RECOMB 2015, Warsaw, Proceedings</a></p>
</li>
<li>
<p><strong>2015-05</strong> | Trans-species learning of cellular signaling systems with bimodal deep belief networks | <em>Lujia Chen, Chunhui Cai, Vicky Chen and Xinghua Lu</em> | <a href="https://doi.org/10.1093/bioinformatics/btv315">Bioinformatics</a></p>
</li>
</ul>
<h3 id="注释基因表达模式-annotating-gene-expression-patterns">注释基因表达模式 （annotating gene expression patterns)</h3>
<ul>
<li><strong>2015-05</strong> | Deep convolutional neural networks for annotating gene expression patterns in the mouse brain | <em>Tao Zeng, Rongjian Li, Ravi Mukkamala, Jieping Ye and Shuiwang Ji</em> | <a href="https://doi.org/10.1186/s12859-015-0553-9">BMC Bioinformatics</a></li>
</ul>
<h3 id="预测-dna-rna-结合蛋白的sequence-specificities">预测 DNA-RNA 结合蛋白的sequence specificities</h3>
<ul>
<li><strong>2015-07</strong> | DeepBind: Predicting the sequence specificities of DNA- and RNA-binding proteins by deep learning | <em>Babak Alipanahi,	 Andrew Delong,	Matthew T. Weirauch &amp; Brendan J. Frey</em> | <a href="https://doi.org/10.1038/nbt.3300">Nature Biotechnology</a></li>
</ul>
<h3 id="调控基因组的综述">调控基因组的综述</h3>
<ul>
<li><strong>2015-08</strong> | Deep learning for regulatory genomics | <em>Yongjin Park &amp; Manolis Kellis</em> | <a href="https://doi.org/10.1038/nbt.3313">Nature Biotechnology</a></li>
</ul>
<h3 id="预测非编码突变nocoding-variants的effects序列模型">预测非编码突变（nocoding variants）的effects【序列模型】</h3>
<ul>
<li><strong>2015-08</strong> | DeepSEA: Predicting effects of noncoding variants with deep learning–based sequence model | <em>Jian Zhou &amp; Olga G. Troyanskaya</em> | <a href="https://doi.org/10.1038/nmeth.3604">Nature Methods: Short intro</a> &amp; <a href="https://doi.org/10.1038/nmeth.3547">Nature Methods</a></li>
</ul>
<h3 id="多平台癌症数据的整合数据分析">多平台癌症数据的整合数据分析</h3>
<ul>
<li><strong>2015-08</strong> | Integrative Data Analysis of Multi-Platform Cancer Data with a Multimodal Deep Learning Approach | <em>Muxuan Liang, Zhizhong Li, Ting Chen, Jianyang Zeng</em> | <a href="https://doi.org/10.1109/TCBB.2014.2377729">IEEE/ACM Transactions on Computational Biology and Bioinformatics (TCBB)</a></li>
</ul>
<h3 id="modeling-structural-features-of-rna-binding-protein-targets">modeling structural features of RNA-binding protein targets</h3>
<ul>
<li><strong>2015-10</strong> | A deep learning framework for modeling structural features of RNA-binding protein targets | <em>Sai Zhang, Jingtian Zhou, Hailin Hu, Haipeng Gong, Ligong Chen, Chao Cheng, and Jianyang Zeng</em> | <a href="https://doi.org/10.1093/nar/gkv1025">NAR</a></li>
</ul>
<h3 id="调控编码regulatory-code">调控编码（regulatory code）</h3>
<ul>
<li><strong>2015-10</strong> | Basset: Learning the regulatory code of the accessible genome with deep convolutional neural networks | <em>David R. Kelley, Jasper Snoek, John Rinn</em> | <a href="https://doi.org/10.1101/028399">Biorxiv</a> | <a href="https://github.com/davek44/Basset">code</a></li>
</ul>
<h3 id="drug-卧槽来鲁华老师的">drug 卧槽来鲁华老师的</h3>
<ul>
<li>
<p><strong>2015-10</strong> | Deep Learning for Drug-Induced Liver Injury | <em>Youjun Xu, Ziwei Dai, Fangjin Chen, Shuaishi Gao, Jianfeng Pei, and Luhua Lai</em> | <a href="https://doi.org/10.1021/acs.jcim.5b00238">ASC Journal of Chemical Information and Modeling</a></p>
</li>
<li>
<p><strong>2016-01</strong> | ADAGE-Based Integration of Publicly Available Pseudomonas aeruginosa Gene Expression Data with Denoising Autoencoders Illuminates Microbe-Host Interactions | <a href="https://dx.doi.org/10.1128/mSystems.00025-15">mSystems</a> | <a href="https://github.com/greenelab/adage">code</a></p>
</li>
<li>
<p><strong>2015-11</strong> | De novo identification of replication-timing domains in the human genome by deep learning | <em>Feng Liu, Chao Ren, Hao Li, Pingkun Zhou, Xiaochen Bo and Wenjie Shu</em> | <a href="https://doi.org/10.1093/bioinformatics/btv643">Bioinformatics</a></p>
</li>
<li>
<p><strong>2015-11</strong> | Recurrent Neural Network Based Hybrid Model of Gene Regulatory Network | <em>Khalid Raza, Mansaf Alam</em> | <a href="https://arxiv.org/abs/1408.5405v2">Arxiv</a></p>
</li>
<li>
<p><strong>2015-11</strong> | Continuous Distributed Representation of Biological Sequences for Deep Proteomics and Genomics | <em>Ehsaneddin Asgari, Mohammad R. K. Mofrad</em> | <a href="http://dx.doi.org/10.1371/journal.pone.0141287">PloS one</a></p>
</li>
<li>
<p><strong>2016-01</strong> | Learning a hierarchical representation of the yeast transcriptomic machinery using an autoencoder model | <em>Lujia Chen, Chunhui Cai, Vicky Chen and Xinghua Lu</em> | <a href="https://doi.org/10.1186/s12859-015-0852-1">BMC Bioinformatics</a></p>
</li>
<li>
<p><strong>2016-01</strong> | PEDLA: predicting enhancers with a deep learning-based algorithmic framework | <em>Feng Liu, Hao Li, Chao Ren, Xiaochen Bo, Wenjie Shu</em> | <a href="https://doi.org/10.1101/036129">Biorxiv</a></p>
</li>
<li>
<p><strong>2016-01</strong> | TensorFlow: Biology’s Gateway to Deep Learning? | <em>Ladislav Rampasek, Anna Goldenberg</em> | <a href="https://doi.org/10.1016/j.cels.2016.01.009">Cell Systems</a></p>
</li>
<li>
<p><strong>2016-01</strong> | ADAGE-Based Integration of Publicly Available Pseudomonas aeruginosa Gene Expression Data with Denoising Autoencoders Illuminates Microbe-Host Interactions | <a href="https://doi.org/10.1128/mSystems.00025-15">mSystems</a> | <a href="https://github.com/greenelab/adage">code</a></p>
</li>
<li>
<p><strong>2016-01</strong> | Deep Learning in Drug Discovery | <em>Erik Gawehn, Jan A. Hiss and Gisbert Schneider</em> | <a href="https://doi.org/10.1002/minf.201501008">Molecular Informatics</a></p>
</li>
<li>
<p><strong>2016-02</strong> | DeepTox: toxicity prediction using deep learning | <em>Andreas Mayr, Günter Klambauer, Thomas Unterthiner, and Sepp Hochreiter</em> | <a href="http://journal.frontiersin.org/article/10.3389/fenvs.2015.00080/full">Frontiers in Environmental Science</a></p>
</li>
<li>
<p><strong>2016-02</strong> | Gene expression inference with deep learning | <em>Yifei Chen, Yi Li, Rajiv Narayan, Aravind Subramanian, Xiaohui Xie</em> | <a href="https://doi.org/10.1093/bioinformatics/btw074">Bioinformatics</a></p>
</li>
<li>
<p><strong>2016-02</strong> | Semi-Supervised Learning of the Electronic Health Record for Phenotype Stratification | <em>Brett Beaulieu-Jones, Casey Greene</em> | <a href="https://doi.org/10.1101/039800">bioRxiv</a></p>
</li>
<li>
<p><strong>2016-03</strong> | Genome-Wide Prediction of cis-Regulatory Regions Using Supervised Deep Learning Methods | <em>Yifeng Li, Wenqiang Shi, Wyeth W Wasserman</em> | <a href="https://doi.org/10.1101/041616">Biorxiv</a></p>
</li>
<li>
<p><strong>2016-03</strong> | Applications of deep learning in biomedicine | <em>Polina Mamoshina, Armando Vieira, Evgeny Putin, and Alex Zhavoronkov</em> | <a href="https://dx.doi.org/10.1021/acs.molpharmaceut.5b00982">ACS Molecular Pharmaceutics</a></p>
</li>
<li>
<p><strong>2016-03</strong> | Deep Learning in Bioinformatics | <em>Seonwoo Min, Byunghan Lee, Sungroh Yoon</em> | <a href="http://arxiv.org/abs/1603.06430">Arxiv</a></p>
</li>
<li>
<p><strong>2016-03</strong> | DeepNano: Deep Recurrent Neural Networks for Base Calling in MinION Nanopore Reads | <em>Vladimír Boža, Broňa Brejová, Tomáš Vinař</em> | <a href="http://arxiv.org/abs/1603.09195">Arxiv</a> | <a href="https://bitbucket.org/vboza/deepnano">code</a></p>
</li>
<li>
<p><strong>2016-03</strong> | deepTarget: End-to-end Learning Framework for microRNA Target Prediction using Deep Recurrent Neural Networks | <em>Byunghan Lee, Junghwan Baek, Seunghyun Park, Sungroh Yoon</em> | <a href="http://arxiv.org/abs/1603.09123">Arxiv</a></p>
</li>
<li>
<p><strong>2016-03</strong> | Deep Learning in Label-free Cell Classification | <em>Claire Lifan Chen, Ata Mahjoubfar, Li-Chia Tai, Ian K. Blaby, Allen Huang, Kayvan Reza Niazi &amp; Bahram Jalali</em> | <a href="https://doi.org/10.1038/srep21471">Nature Scientific Reports</a></p>
</li>
<li>
<p><strong>2016-04</strong> | Accurate classification of protein subcellular localization from high throughput microscopy images using deep learning | <em>Tanel Pärnamaa, Leopold Parts</em> | <a href="http://dx.doi.org/10.1101/050757">bioRxiv</a></p>
</li>
<li>
<p><strong>2016-04</strong> | DanQ: a hybrid convolutional and recurrent deep neural network for quantifying the function of DNA sequences | <em>Daniel Quang &amp; Xiaohui Xie</em> | <a href="https://doi.org/10.1093/nar/gkw226">Nucleic Acids Research</a> | <a href="https://github.com/uci-cbcl/DanQ">code</a></p>
</li>
<li>
<p><strong>2016-04</strong> | deepMiRGene: Deep Neural Network based Precursor microRNA Prediction | <em>Seunghyun Park, Seonwoo Min, Hyun-soo Choi, and Sungroh Yoon</em> | <a href="http://arxiv.org/abs/1605.00017">Arxiv</a></p>
</li>
<li>
<p><strong>2016-04</strong> | Microscopy cell counting and detection with fully convolutional regression networks | <em>Weidi Xie, J. Alison Noble and Andrew Zisserman</em> | <a href="https://doi.org/10.1080/21681163.2016.1149104">Computer Methods in Biomechanics and Biomedical Engineering: Imaging &amp; Visualization</a></p>
</li>
<li>
<p><strong>2016-04</strong> | Protein Secondary Structure Prediction Using Cascaded Convolutional and Recurrent Neural Networks | <em>Zhen Li and Yizhou Yu</em> | <a href="https://arxiv.org/abs/1604.07176">Arxiv</a></p>
</li>
<li>
<p><strong>2016-05</strong> | Denoising genome-wide histone ChIP-seq with convolutional neural networks | <em>Pang Wei Koh, Emma Pierson, Anshul Kundaje</em> | <a href="https://doi.org/10.1101/052118">Biorxiv</a></p>
</li>
<li>
<p><strong>2016-05</strong> | Deep Motif: Visualizing Genomic Sequence Classifications | <em>Jack Lanchantin, Ritambhara Singh, Zeming Lin, Yanjun Qi</em> | <a href="http://arxiv.org/abs/1605.01133">Arxiv</a></p>
</li>
<li>
<p><strong>2016-05</strong> | Not Just a Black Box: Learning Important Features Through Propagating Activation Differences | <em>Avanti Shrikumar, Peyton Greenside, Anna Shcherbina, Anshul Kundaje</em> | <a href="https://arxiv.org/abs/1605.01713">Arxiv</a></p>
</li>
<li>
<p><strong>2016-05</strong> | Deep biomarkers of human aging: Application of deep neural networks to biomarker development | <em>Evgeny Putin, Polina Mamoshina, Alexander Aliper, Mikhail Korzinkin, Alexey Moskalev, Alexey Kolosov, Alexander Ostrovskiy, Charles Cantor, Jan Vijg, and Alex Zhavoronkov</em> | <a href="https://doi.org/10.18632/aging.100968">Aging</a></p>
</li>
<li>
<p><strong>2016-05</strong> | Deep learning applications for predicting pharmacological properties of drugs and drug repurposing using transcriptomic data | <em>Alexander Aliper, Sergey Plis, Artem Artemov, Alvaro Ulloa, Polina Mamoshina, and Alex Zhavoronkov</em> | <a href="https://doi.org/10.1021/acs.molpharmaceut.6b00248">ACS Molecular Pharmaceutics</a></p>
</li>
<li>
<p><strong>2016-05</strong> | Deep Machine Learning provides state-of-the-art performance in image-based plant phenotyping | <em>Michael P. Pound, Alexandra J. Burgess, Michael H. Wilson, Jonathan A. Atkinson, Marcus Griffiths, Aaron S. Jackson, Adrian Bulat, Yorgos Tzimiropoulos, Darren M. Wells, Erik H. Murchie, Tony P. Pridmore, Andrew P. French</em> | <a href="https://doi.org/10.1101/053033">Biorxiv</a></p>
</li>
<li>
<p><strong>2016-05</strong> | Genetic Architect: Discovering Genomic Structure with Learned Neural Architectures | <em>Laura Deming, Sasha Targ, Nate Sauder, Diogo Almeida, Chun Jimmie Ye</em> | <a href="https://arxiv.org/abs/1605.07156v1">Arxiv</a></p>
</li>
<li>
<p><strong>2016-05</strong> | DeepCyTOF: Automated Cell Classification of Mass Cytometry Data by Deep Learning and Domain Adaptation | <em>Huamin Li, Uri Shaham, Yi Yao, Ruth Montgomery, Yuval Kluger</em> | <a href="https://doi.org/10.1101/054411">Biorxiv</a></p>
</li>
<li>
<p><strong>2016-06</strong> | Classifying and segmenting microscopy images with deep multiple instance learning | <em>Oren Z. Kraus, Jimmy Lei Ba and Brendan J. Frey</em> | <a href="https://doi.org/10.1093/bioinformatics/btw252">Bioinformatics</a></p>
</li>
<li>
<p><strong>2016-06</strong> | Convolutional neural network architectures for predicting DNA–protein binding | <em>Haoyang Zeng, Matthew D. Edwards, Ge Liu and David K. Gifford</em>  | <a href="https://doi.org/10.1093/bioinformatics/btw255">Bioinformatics</a> | <a href="http://cnn.csail.mit.edu">code</a></p>
</li>
<li>
<p><strong>2016-06</strong> | DeepLNC, a long non-coding RNA prediction tool using deep neural network | <em>Rashmi Tripathi, Sunil Patel, Vandana Kumari, Pavan Chakraborty, Pritish Kumar Varadwaj</em> | <a href="https://doi.org/10.1007/s13721-016-0129-2">Network Modeling Analysis in Health Informatics and Bioinformatics</a></p>
</li>
<li>
<p><strong>2016-06</strong> | Virtual Screening: A Challenge for Deep Learning | <em>Javier Pérez-Sianes, Horacio Pérez-Sánchez, Fernando Díaz</em> | <a href="https://doi.org/10.1007/978-3-319-40126-3_2">10th International Conference on Practical Applications of Computational Biology &amp; Bioinformatics</a></p>
</li>
<li>
<p><strong>2016-07</strong> | Deep learning for computational biology | <em>Christof Angermueller, Tanel Pärnamaa, Leopold Parts, Oliver Stegle</em> | <a href="https://doi.org/10.15252/msb.20156651">Molecular Systems Biology</a></p>
</li>
<li>
<p><strong>2016-07</strong> | Deep Learning in Bioinformatics | <em>Seonwoo Min, Byunghan Lee, Sungroh Yoon</em> | <a href="https://doi.org/10.1093/bib/bbw068">Briefings in Bioinformatics</a></p>
</li>
<li>
<p><strong>2016-08</strong> | DeepChrome: deep-learning for predicting gene expression from histone modifications | <em>Ritambhara Singh, Jack Lanchantin,  Gabriel Robins,  Yanjun Qi</em> | <a href="https://doi.org/10.1093/bioinformatics/btw427">Bioinformatics</a></p>
</li>
<li>
<p><strong>2016-08</strong> | Deep Artificial Neural Networks and Neuromorphic Chips for Big Data Analysis: Pharmaceutical and Bioinformatics Applications | <em>Lucas Antón Pastur-Romay, Francisco Cedrón, Alejandro Pazos and Ana Belén Porto-Pazos</em> | <a href="https://doi.org/10.3390/ijms17081313">International Journal of Molecular Sciences</a></p>
</li>
<li>
<p><strong>2016-08</strong> | Deep GDashboard: Visualizing and Understanding Genomic Sequences Using Deep Neural Networks | <em>Jack Lanchantin, Ritambhara Singh, Beilun Wang, Yanjun Qi</em> | <a href="https://arxiv.org/abs/1608.03644v2">Arxiv</a></p>
</li>
<li>
<p><strong>2016-08</strong> | Modeling translation elongation dynamics by deep learning reveals new insights into the landscape of ribosome stalling | <em>Sai Zhang, Hailin Hu, Jingtian Zhou, Xuan He and Jianyang Zeng</em> | <a href="http://dx.doi.org/10.1101/067108">bioRxiv</a></p>
</li>
<li>
<p><strong>2016-08</strong> | DeepWAS: Directly integrating regulatory information into GWAS using deep learning supports master regulator MEF2C as risk factor for major depressive disorder | <em>Gökcen Eraslan, Janine Arloth, Jade Martins, Stella Iurato, Darina Czamara, Elisabeth B. Binder, Fabian J. Theis, Nikola S. Mueller</em> | <a href="https://dx.doi.org/10.1101/069096">bioRxiv</a></p>
</li>
<li>
<p><strong>2016-09</strong> | The Next Era: Deep Learning in Pharmaceutical Research | <em>Sean Ekins</em> | <a href="https://dx.doi.org/10.1007/s11095-016-2029-7">Pharmaceutical Research</a></p>
</li>
<li>
<p><strong>2016-10</strong> | Automatic chemical design using a data-driven continuous representation of molecules | <em>Rafael Gómez-Bombarelli, David Duvenaud, José Miguel Hernández-Lobato, Jorge Aguilera-Iparraguirre, Timothy D. Hirzel, Ryan P. Adams, Alán Aspuru-Guzik</em> | <a href="https://arxiv.org/abs/1610.02415">Arxiv</a></p>
</li>
<li>
<p><strong>2016-10</strong> | FIDDLE: An integrative deep learning framework for functional genomic data inference | <em>Umut Eser, L. Stirling Churchman</em> | <a href="http://dx.doi.org/10.1101/081380">bioRxiv</a></p>
</li>
<li>
<p><strong>2016-10</strong> | Deep Learning for Imaging Flow Cytometry: Cell Cycle Analysis of Jurkat Cells | <em>Philipp Eulenberg, Niklas Koehler, Thomas Blasi, Andrew Filby, Anne E. Carpenter, Paul Rees, Fabian J. Theis, F. Alexander Wolf</em> | <a href="http://dx.doi.org/10.1101/081364">bioRxiv</a></p>
</li>
<li>
<p><strong>2016-10</strong> | Leveraging uncertainty information from deep neural networks for disease detection | <em>Christian Leibig, Vaneeda Allken, Philipp Berens, Siegfried Wahl</em> | <a href="http://dx.doi.org/10.1101/084210">bioRxiv</a></p>
</li>
<li>
<p><strong>2016-11</strong> | Predicting Enhancer-Promoter Interaction from Genomic Sequence with Deep Neural Networks | <em>Shashank Singh, Yang Yang, Barnabas Poczos, Jian Ma</em> | <a href="https://doi.org/10.1101/085241">bioRxiv</a></p>
</li>
<li>
<p><strong>2016-11</strong> | RNA-protein binding motifs mining with a new hybrid deep learning based cross-domain knowledge integration approach | <em>Xiaoyong Pan, Hong-Bin Shen</em> | <a href="http://dx.doi.org/10.1101/085191">bioRxiv</a></p>
</li>
<li>
<p><strong>2016-11</strong> | Low Data Drug Discovery with One-shot Learning | <em>Han Altae-Tran, Bharath Ramsundar, Aneesh S. Pappu, Vijay Pande</em> | <a href="https://arxiv.org/abs/1611.03199">Arxiv</a></p>
</li>
<li>
<p><strong>2016-11</strong> | Diet Networks: Thin Parameters for Fat Genomic | <em>Adriana Romero, Pierre Luc Carrier, Akram Erraqabi, Tristan Sylvain, Alex Auvolat, Etienne Dejoie, Marc-André Legault, Marie-Pierre Dubé, Julie G. Hussin, Yoshua Bengio</em> | <a href="https://arxiv.org/abs/1611.09340">Arxiv</a></p>
</li>
<li>
<p><strong>2016-11</strong> | DeeperBind: Enhancing Prediction of Sequence Specificities of DNA Binding Proteins | <em>Hamid Reza Hassanzadeh, May D. Wang</em> | <a href="https://arxiv.org/abs/1611.05777">Arxiv</a></p>
</li>
<li>
<p><strong>2016-11</strong> | Deep learning with feature embedding for compound-protein interaction prediction | <em>Fangping Wan, Jianyang Zeng</em> | <a href="https://doi.org/10.1101/086033">bioRxiv</a></p>
</li>
<li>
<p><strong>2016-11</strong> | Deep Learning Automates the Quantitative Analysis of Individual Cells in Live-Cell Imaging Experiments | <em>David A. Van Valen, Takamasa Kudo, Keara M. Lane, Derek N. Macklin, Nicolas T. Quach, Mialy M. DeFelice, Inbal Maayan, Yu Tanouchi, Euan A. Ashley, Markus W. Covert</em> | <a href="https://doi.org/10.1371/journal.pcbi.1005177">PLoS Computational Biology</a></p>
</li>
<li>
<p><strong>2016-12</strong> | Creating a universal SNP and small indel variant caller with deep neural networks | <em>Ryan Poplin, Dan Newburger, Jojo Dijamco, Nam Nguyen, Dion Loy, Sam S. Gross, Cory Y. McLean, Mark A. DePristo</em> | <a href="https://doi.org/10.1101/092890">bioRxiv</a></p>
</li>
<li>
<p><strong>2016-12</strong> | DeepCancer: Detecting Cancer through Gene Expressions via Deep Generative Learning | <em>Rajendra Rana Bhat, Vivek Viswanath, Xiaolin Li</em> | <a href="http://arxiv.org/abs/1612.03211">Arxiv</a></p>
</li>
<li>
<p><strong>2016-12</strong> | Cox-nnet: an artificial neural network Cox regression for prognosis prediction | <em>Travers Ching, Xun Zhu, Lana Garmire</em> | <a href="https://doi.org/10.1101/093021">bioRxiv</a></p>
</li>
<li>
<p><strong>2016-12</strong> | Deep learning is effective for the classification of OCT images of normal versus Age-related Macular Degeneration | <em>Cecilia S Lee, Doug M Baughman, Aaron Y Lee</em> | <a href="https://doi.org/10.1101/094276">bioRxiv</a></p>
</li>
<li>
<p><strong>2016-12</strong> | Partitioned learning of deep Boltzmann machines for SNP data | <em>Moritz Hess, Stefan Lenz, Tamara Blaette, Lars Bullinger, Harald Binder</em> | <a href="https://doi.org/10.1101/095638">bioRxiv</a></p>
</li>
<li>
<p><strong>2016-12</strong> | DeepAD: Alzheimer′s Disease Classification via Deep Convolutional Neural Networks using MRI and fMRI | <em>Saman Sarraf, John Anderson, Ghassem Tofighi, for the Alzheimer's Disease Neuroimaging Initiativ</em> | <a href="https://doi.org/10.1101/070441">bioRxiv</a></p>
</li>
<li>
<p><strong>2016-12</strong> | Training Genotype Callers with Neural Networks | <em>Rémi Torracinta, Fabien Campagne</em> | <a href="https://doi.org/10.1101/097469">bioRxiv</a></p>
</li>
<li>
<p><strong>2016-12</strong> | EP-DNN: A Deep Neural Network-Based Global Enhancer Prediction Algorithm | <em>Seong Gon Kim, Mrudul Harwani, Ananth Grama, Somali Chaterji</em> | <a href="https://doi.org/10.1038/srep38433">Nature Scientific Reports</a></p>
</li>
<li>
<p><strong>2016-12</strong> | EnhancerPred: a predictor for discovering enhancers based on the combination and selection of multiple features | <em>Cangzhi Jia, Wenying He</em> | <a href="https://doi.org/10.1038/srep38741">Nature Scientific Reports</a></p>
</li>
<li>
<p><strong>2016-12</strong> | DeepEnhancer: Predicting enhancers by convolutional neural networks | <em>Min, Xu, Ning Chen, Ting Chen, and Rui Jiang</em> | <a href="https://doi.org/10.1109/BIBM.2016.7822593">2016 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</a></p>
</li>
<li>
<p><strong>2016-12</strong> | DeepSplice: Deep classification of novel splice junctions revealed by RNA-seq | <em>Zhang, Yi, Xinan Liu, James N. MacLeod, and Jinze Liu</em> | <a href="https://doi.org/10.1109/BIBM.2016.7822541">2016 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</a></p>
</li>
<li>
<p><strong>2016-12</strong> | Deep convolutional neural networks for detecting secondary structures in protein density maps from cryo-electron microscopy | <em>Li, Rongjian, Dong Si, Tao Zeng, Shuiwang Ji, and Jing He</em> | <a href="https://doi.org/10.1109/BIBM.2016.7822490">2016 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</a></p>
</li>
<li>
<p><strong>2016-12</strong> | Towards recognition of protein function based on its structure using deep convolutional networks | <em>Tavanaei, Amirhossein, Anthony S. Maida, Arun Kaniymattam, and Rasiah Loganantharaj</em> | <a href="https://doi.org/10.1109/BIBM.2016.7822509">2016 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</a></p>
</li>
<li>
<p><strong>2016-12</strong> | Emotion recognition from multi-channel EEG data through Convolutional Recurrent Neural Network | <em>Li, Xiang, Dawei Song, Peng Zhang, Guangliang Yu, Yuexian Hou, and Bin Hu</em> | <a href="https://doi.org/10.1109/BIBM.2016.7822545">2016 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</a></p>
</li>
<li>
<p><strong>2016-12</strong> | Coarse-to-Fine Stacked Fully Convolutional Nets for lymph node segmentation in ultrasound images | <em>Zhang, Yizhe, Michael TC Ying, Lin Yang, Anil T. Ahuja, and Danny Z. Chen</em> | <a href="https://doi.org/10.1109/BIBM.2016.7822557">2016 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</a></p>
</li>
<li>
<p><strong>2016-12</strong> | CNNsite: Prediction of DNA-binding residues in proteins using Convolutional Neural Network with sequence features | <em>Zhou, Jiyun, Qin Lu, Ruifeng Xu, Lin Gui, and Hongpeng Wang</em> | <a href="https://doi.org/10.1109/BIBM.2016.7822496">2016 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</a></p>
</li>
<li>
<p><strong>2016-12</strong> | A predictive model of gene expression using a deep learning framework | <em>Xie, Rui, Andrew Quitadamo, Jianlin Cheng, and Xinghua Shi</em> | <a href="https://doi.org/10.1109/BIBM.2016.7822599">2016 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</a></p>
</li>
<li>
<p><strong>2016-12</strong> | Deep convolutional neural network for survival analysis with pathological images | <em>Zhu, Xinliang, Jiawen Yao, and Junzhou Huang</em> | <a href="https://doi.org/10.1109/BIBM.2016.7822579">2016 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</a></p>
</li>
<li>
<p><strong>2016-12</strong> | Dependency-based convolutional neural network for drug-drug interaction extraction | <em>Liu, Shengyu, Kai Chen, Qingcai Chen, and Buzhou Tang</em> | <a href="https://doi.org/10.1109/BIBM.2016.7822671">2016 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</a></p>
</li>
<li>
<p><strong>2016-12</strong> | Pervasive EEG diagnosis of depression using Deep Belief Network with three-electrodes EEG collector | <em>Cai, Hanshu, Xiaocong Sha, Xue Han, Shixin Wei, and Bin Hu</em> | <a href="https://doi.org/10.1109/BIBM.2016.7822696">2016 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</a></p>
</li>
<li>
<p><strong>2016-12</strong> | Cardiac left ventricular volumes prediction method based on atlas location and deep learning | <em>Luo, Gongning, Suyu Dong, Kuanquan Wang, and Henggui Zhang</em> | <a href="https://doi.org/10.1109/BIBM.2016.7822759">2016 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</a></p>
</li>
<li>
<p><strong>2016-12</strong> | A high-precision shallow Convolutional Neural Network based strategy for the detection of Genomic Deletions | <em>Wang, Jing, Cheng Ling, and Jingyang Gao</em> | <a href="https://doi.org/10.1109/BIBM.2016.7822793">2016 IEEE International Conference on Bioinformatics and Biomedicine (BIBM)</a></p>
</li>
<li>
<p><strong>2016-12</strong> | The cornucopia of meaningful leads: Applying deep adversarial autoencoders for new molecule development in oncology | <em>Kadurin, Artur, Alexander Aliper, Andrey Kazennov, Polina Mamoshina, Quentin Vanhaelen, Kuzma Khrabrov, and Alex Zhavoronkov</em> | <a href="https://doi.org/10.18632/oncotarget.14073">Oncotarget</a></p>
</li>
<li>
<p><strong>2016-12</strong> | Medical Image Synthesis with Context-Aware Generative Adversarial Networks | <em>Dong Nie, Roger Trullo, Caroline Petitjean, Su Ruan, Dinggang Shen</em> | <a href="https://arxiv.org/abs/1612.05362">Arxiv</a></p>
</li>
<li>
<p><strong>2016-12</strong> | Unsupervised Learning from Noisy Networks with Applications to Hi-C Data | <em>Wang, Bo, Junjie Zhu, Armin Pourshafeie, Oana Ursu, Serafim Batzoglou, and Anshul Kundaje</em> | <a href="http://papers.nips.cc/paper/6291-unsupervised-learning-from-noisy-networks-with-applications-to-hi-c-data">Advances in Neural Information Processing Systems (NIPS 2016)</a></p>
</li>
<li>
<p><strong>2016-12</strong> | Deep Learning for Health Informatics | <em>Daniele Ravì, Charence Wong, Fani Deligianni, Melissa Berthelot, Javier Andreu-Perez, Benny Lo, and Guang-Zhong Yang</em> | <a href="https://doi.org/10.1109/JBHI.2016.2636665">IEEE Journal of Biomedical and Health Informatics</a></p>
</li>
<li>
<p><strong>2017-01</strong> | A Deep Learning Approach for Cancer Detection and Relevant Gene Identification | <em>Wang, Jing, Cheng Ling, and Jingyang Gao</em> | <a href="http://dx.doi.org/10.1142/9789813207813_0022">Pacific Symposium on Biocomputing 2017</a></p>
</li>
<li>
<p><strong>2017-01</strong> | Deep Motif Dashboard: Visualizing and Understanding Genomic Sequences Using Deep Neural Networks | <em>Lanchantin, Jack, Ritambhara Singh, Beilun Wang, and Yanjun Qi</em> | <a href="http://dx.doi.org/10.1142/9789813207813_0025">Pacific Symposium on Biocomputing 2017</a></p>
</li>
<li>
<p><strong>2017-01</strong> | HLA class I binding prediction via convolutional neural networks | <em>Yeeleng Scott Vang, Xiaohui Xie</em> | <a href="https://doi.org/10.1101/099358">bioRxiv</a></p>
</li>
<li>
<p><strong>2017-01</strong> | DeadNet: Identifying Phototoxicity from Label-free Microscopy Images of Cells using Deep ConvNets | <em>David Richmond, Anna Payne-Tobin Jost, Talley Lambert, Jennifer Waters, Hunter Elliott</em> | <a href="https://arxiv.org/abs/1701.06109">arXiv</a></p>
</li>
<li>
<p><strong>2017-01</strong> | Dermatologist-level classification of skin cancer with deep neural networks | <em>Andre Esteva, Brett Kuprel, Roberto A. Novoa, Justin Ko, Susan M. Swetter, Helen M. Blau &amp; Sebastian Thrun</em> | <a href="https://doi.org/10.1038/nature21056">Nature</a></p>
</li>
<li>
<p><strong>2017-01</strong> | Understanding sequence conservation with deep learning | <em>Yi Li, Daniel Quang, Xiaohui Xie</em> | <a href="https://doi.org/10.1101/103929">Biorxiv</a></p>
</li>
<li>
<p><strong>2017-01</strong> | Learning the Structural Vocabulary of a Network | <em>Saket Navlakha</em> | <a href="https://doi.org/10.1162/NECO_a_00924">Neural Computation</a></p>
</li>
<li>
<p><strong>2017-01</strong> | Mining the Unknown: Assigning Function to Noncoding Single Nucleotide Polymorphisms | <em>Sierra S. Nishizaki, Alan P. Boyle</em> | <a href="http://dx.doi.org/10.1016/j.tig.2016.10.008">Trends in Genetics</a></p>
</li>
<li>
<p><strong>2017-01</strong> | Reverse-complement parameter sharing improves deep learning models for genomics | <em>Avanti Shrikumar, Peyton Greenside, Anshul Kundaje</em> | <a href="https://doi.org/10.1101/103663">bioRxiv</a></p>
</li>
<li>
<p><strong>2017-01</strong> | TIDE: predicting translation initiation sites by deep learning | <em>Sai Zhang, Hailin Hu, Tao Jiang, Lei Zhang, Jianyang Zeng</em> | <a href="https://doi.org/10.1101/103374">bioRxiv</a></p>
</li>
<li>
<p><strong>2017-01</strong> | Integrative Deep Models for Alternative Splicing | <em>Anupama Jha, Matthew R Gazzara, Yoseph Barash</em> | <a href="https://doi.org/10.1101/104869">bioRxiv</a></p>
</li>
<li>
<p><strong>2017-01</strong> | Deep Recurrent Neural Network for Protein Function Prediction from Sequence | <em>Xueliang Leon Liu</em> | <a href="https://doi.org/10.1101/103994">bioRxiv</a></p>
</li>
<li>
<p><strong>2017-01</strong> | Nucleotide sequence and DNaseI sensitivity are predictive of 3D chromatin architecture | <em>Jacob Schreiber, Maxwell Libbrecht, Jeffrey Bilmes, William Noble</em> | <a href="https://doi.org/10.1101/103614">bioRxiv</a></p>
</li>
<li>
<p><strong>2017-01</strong> | Accurate De Novo Prediction of Protein Contact Map by Ultra-Deep Learning Model | <em>Sheng Wang, Siqi Sun, Zhen Li, Renyu Zhang, Jinbo Xu</em> | <a href="https://doi.org/10.1371/journal.pcbi.1005324">PloS Computational Biology</a></p>
</li>
<li>
<p><strong>2017-02</strong> | Imputation for transcription factor binding predictions based on deep learning | <em>Qian Qin, Jianxing Feng</em> | <a href="http://dx.doi.org/10.1371/journal.pcbi.1005403">PloS Computational Biology</a></p>
</li>
<li>
<p><strong>2017-02</strong> | Deep Learning based multi-omics integration robustly predicts survival in liver cancer | <em>Kumardeep Chaudhary, Olivier B. Poirion, Liangqun Lu, Lana Garmire</em> | <a href="https://doi.org/10.1101/114892">bioRxiv</a></p>
</li>
<li>
<p><strong>2017-03</strong> | Predicting the impact of non-coding variants on DNA methylation | <em>Zeng, Haoyang, and David K. Gifford</em> | <a href="https://doi.org/10.1093/nar/gkx177">Nucleic Acids Research</a></p>
</li>
<li>
<p><strong>2017-03</strong> | H&amp;E-stained Whole Slide Image Deep Learning Predicts SPOP Mutation State in Prostate Cancer | <em>Andrew J Schaumberg, Mark A Rubin, Thomas J Fuchs</em> | <a href="https://doi.org/10.1101/064279">bioRxiv</a></p>
</li>
<li>
<p><strong>2017-04</strong> | DeepCpG: accurate prediction of single-cell DNA methylation states using deep learning | <em>Christof Angermueller, Heather J. Lee, Wolf Reik, and Oliver Stegle</em> | <a href="https://doi.org/10.1186/s13059-017-1189-z">Genome Biology</a></p>
</li>
<li>
<p><strong>2017-04</strong> | Generalising Better: Applying Deep Learning To Integrate Deleteriousness Prediction Scores For Whole-Exome SNV Studies | <em>Ilia Korvigo, Andrey Afanasyev, Nikolay Romashchenko, Mihail Skoblov</em> | <a href="https://doi.org/10.1101/126532">bioRxiv</a></p>
</li>
<li>
<p><strong>2017-09</strong> | DeepLoc: prediction of protein subcellular localization using deep learning | <em>José JA Armenteros, Casper K Sønderby, Søren K Sønderby, Henrik Nielsen, Ole Winther</em> | <a href="https://doi.org/10.1093/bioinformatics/btx431">Bioinformatics</a></p>
</li>
<li>
<p><strong>2017-11</strong> | Modeling positional effects of regulatory sequences with spline transformations increases prediction accuracy of deep neural networks | <em>Žiga Avsec, Mohammadamin Barekatain, Jun Cheng, Julien Gagneur</em> | <a href="https://doi.org/10.1093/bioinformatics/btx727">Bioinformatics</a></p>
</li>
<li>
<p><strong>2017-11</strong> | Protein Loop Modeling Using Deep Generative Adversarial Network | <em>Zhaoyu Li, Son P. Nguyen, Dong Xu, Yi Shang</em> | <a href="https://ieeexplore.ieee.org/abstract/document/8372069">ICTAI</a></p>
</li>
<li>
<p><strong>2017-12</strong> | Variational auto-encoding of protein sequences | <em>Sam Sinai, Eric Kelsic, George M. Church and Martin A. Nowak</em> | <a href="https://arxiv.org/abs/1712.03346">arxiv</a></p>
</li>
<li>
<p><strong>2017-12</strong> | Predicting enhancers with deep convolutional neural networks | <em>Xu Min, Wanwen Zeng, Shengquan Chen, Ning Chen, Ting Chen and Rui Jiang</em> | <a href="https://doi.org/10.1186/s12859-017-1878-3">BMC Bioinformatics</a></p>
</li>
<li>
<p><strong>2018-06</strong> | Convolutional neural networks for classification of alignments of non-coding RNA sequences | <em>Genta Aoki, Yasubumi Sakakibara</em> | <a href="https://doi.org/10.1093/bioinformatics/bty228">Bioinformatics</a></p>
</li>
<li>
<p><strong>2019-02</strong> | DeepRibo: a neural network for precise gene annotation of prokaryotes by combining ribosome profiling signal and binding site patterns | <em>Jim Clauwaert, Gerben Menschaert, Willem Waegeman</em> | <a href="https://doi.org/10.1093/nar/gkz061">Nucleic Acids Research</a></p>
</li>
</ul>

                </div>
            </article>
        </div>

        
            <div class="next-post">
                <div class="next gt-c-content-color-first">下一篇</div>
                <a href="https://justindoit.github.io/post/hAF34VF4V/" class="post-title gt-a-link">
                    一年思考（一）科研素养
                </a>
            </div>
        

        

        

        

        <div class="site-footer gt-c-content-color-first">
    <div class="slogan gt-c-content-color-first">满招损，谦得益</div>
    <div class="social-container">
        
            
        
            
        
            
        
            
        
            
        
            
        
    </div>
    <div class="footer-info">
        Powered by <a href="https://github.com/getgridea/gridea" target="_blank">Gridea</a>
    </div>
    <div>
        Theme by <a href="https://imhanjie.com/" target="_blank">imhanjie</a>, Powered by <a
                href="https://github.com/getgridea/gridea" target="_blank">Gridea | <a href="https://justindoit.github.io/atom.xml" target="_blank">RSS</a></a>
    </div>
</div>

<script>
  hljs.initHighlightingOnLoad()
</script>

    </div>
</div>
</body>
</html>
